### PR TITLE
Fix #271 Update the initial used for yandex which clashed with yahoo

### LINKED ISSUE #271: Yandex search uses Yahoo, not Yandex
**I'm submitting a ...**
- [X] bug report
- [ ] feature request

**Current behavior:**

The Yandex button runs a Yahoo search.

**Expected behavior:**

The Yandex button runs a Yandex search.

**Steps to reproduce:**
Enter a search term and click the Yandex button.

### PR DESCRIPTION
**Pull request by raju249**
Fixes #271 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Quick fix, update initials used for yandex that clashed with yahoo.

### CODE REVIEW

**Comment by raju249** [comment]
That is taken care off in other PR. Only problem all the other projects need to be sent an update to work on their calls. 
Since changing it would return them 404 otherwise.

File: app/scrapers/__init__.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
     'y': Yahoo(),
     'd': Duckduckgo(),
     'a': Ask(),
-    'yd': Yandex(),
+    'n': Yandex(),
```
Comment (AnshulMalik) [inline app/scrapers/__init__.py]: we should just use the full names instead of just one or two chars here

Comment (raju249) [reply app/scrapers/__init__.py]: Yes @AnshulMalik 
We have an issue where in we are going to change this initials setup and use the whole name.
This PR temporarily takes care of the yandex using yahoo.

**Comment by cclauss** [comment]
We should use full names instead of initials.

File: app/scrapers/__init__.py (hunk 1 above)
Comment (AnshulMalik) [reply app/scrapers/__init__.py]: ok cool

**Comment by raju249** [comment]
Cool then.
Will change it to use full names.
:+1

**Comment by AnshulMalik** [comment]
Hey, https://github.com/fossasia/query-server/pull/377 has a nice broade fix :)

**Comment by raju249** [comment]
Closing this as its taken care of in #377

**Pull request closed without merging**